### ISSUE TITLE
Environment file sync subcommand

### ISSUE THREAD

**Issue by drollinger** [denoland/deployctl#138]
I'm hoping this fits in the scope of this cmd line tool. 

It would be helpful to have capability to send my local environment .env file to my project and have it save all the variables defined. This way when developing locally, I can get my environment defined in a file which is easier to work with then having to enter in fields that hide values once saved. Then I can simply send it to a project and have variables created or updated. This would also be beneficial for reference when building out tools to automate rolling credentials. 

I did build out this functionality already on my fork so if it is of any interest, I can clean it up and submit a pull request.

**Comment by arnauorriols** [comment]
Hey @drollinger. There's some work going on around env handling (see #179). The feature you've implemented in your fork is definitely interesting. if you are still up for it, you are welcome to submit a pull request!

**Comment by drollinger** [comment]
@arnauorriols Yeah, I can look things over again and submit a PR, this was from awhile ago so it may take a sec to remember how things where set up.
